pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -161,7 +163,16 @@ func (ka *kubeactions) List(ctx context.Context, oc *api.OpenShiftCluster, group
 		return nil, err
 	}
 
-	return ul.MarshalJSON()
+	if jpath == nil {
+		return ul.MarshalJSON()
+	}
+
+	results, err := jpath.Search(ul)
```

Comment:
this nearly works but not quite.  If I query `items[0]` I see:

```
{
  "Object": {
     ...
  }
}
```

and I should see

```
{
   ...
}
```

The fix, unpleasant as it is, is to do:

```
	b, err := ul.MarshalJSON()
	if err != nil {
		return err
	}

	var o interface{}
	err = json.Unmarshal(b, &o)
	if err != nil {
		return err
	}

	o, err = jpath.Search(o)
	if err != nil {
		return err
	}

	return json.Marshal(o)
```

Reply:
should be good now..